grabbit/src/main/groovy/com/twcable/grabbit/resources/GrabbitResourceProvider.groovy

```diff
@@ -72,14 +61,16 @@ class GrabbitResourceProvider implements ResourceProvider {
     @Override
     Resource getResource(ResourceResolver resolver, String path) {
         switch (path) {
-            case ~/^${GRABBIT_JOB}$/:
-                log.debug "Called for path : ${path}"
-                return getJobResource(resolver, path, GRABBIT_JOB_RESOURCE_TYPE)
-            case ~/^${GRABBIT_JOB}\/(.+).(html|json)$/:
-                log.debug "Called for path : ${path}"
-                return getJobResource(resolver, path, GRABBIT_JOB_RESOURCE_TYPE)
+            case ~/^\/grabbit\/job(\/.*)?$/:
+                log.debug "Resolving ${path} to JobResource"
+                return new JobResource(resolver, path)
+            case ~/^\/grabbit\/transaction(\/.*)?$/:
+                log.debug "Resolving ${path} to TransactionResource"
+                return new TransactionResource(resolver, path)
+            case ~/^\/grabbit\/content(\/)?$/:
+                log.debug "Resolving ${path} to ContentResource"
+                return new ContentResource(resolver, path)
             default:
-                log.warn "Unable to find resource for path: ${path}."
```

Comment:
We are seeing this warning all over the place, "Unable to find resource for path: /grabbit". In the case any Grabbit resource is queried but unable to resolve (common case is for unauthorized access), Sling tries the parent resource (in this case /grabbit). It's a scary message to some, and really isn't useful; so removing it.